assets/js/ecoindex-badge.js

```diff
@@ -11,8 +11,8 @@
     a.title = "Analyse ecoindex";
     img.src = `${baseUrl}/badge/?theme=${theme}&url=${url}`;
     img.alt = "Badge ecoindex";
-    img.width = "108px";
-    img.height = "32px";
+    img.setAttribute('width', "108px");
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it’s relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->

**❓ question(non-blocking):** ne faut il pas uniformiser avec des single quote ?

Reply:
^^
Si, ce serait mieux